<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1262</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/new/prefix1/Data/Qt_logo.ico</normaloff>:/new/prefix1/Data/Qt_logo.ico</iconset>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>831</width>
     <height>221</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
  </widget>
  <widget class="QGroupBox" name="GB_SIze">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>831</width>
     <height>101</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="title">
    <string>字体</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QCheckBox" name="Check_Underline">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Underline</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Check_Italic">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Italic</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Check_Bold">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Bold</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GB_Color">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>831</width>
     <height>111</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>颜色</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QRadioButton" name="radio_Bt_Black">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Black</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_Bt_Red">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Red</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_Bt_Blue">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>60</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Bt_Clear">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>490</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>CLear</string>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/prefix1/Data/Exit.jpg</normaloff>:/new/prefix1/Data/Exit.jpg</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_OK">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>490</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_Close">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>490</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Bt_Close</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>980</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Bt_Clear</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>625</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Bt_OK</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>645</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
